plugins/ua_nodestore_ziptree.c

```diff
@@ -4,9 +4,11 @@
  *    Copyright 2014-2018 (c) Fraunhofer IOSB (Author: Julius Pfrommer)
  *    Copyright 2017 (c) Julian Grothoff
  *    Copyright 2017 (c) Stefan Profanter, fortiss GmbH
+ *    Copyright 2020 (c) Kalycito Infotech Pvt Ltd
  */
 
 #include <open62541/plugin/nodestore_default.h>
+#include <open62541/plugin/log_stdout.h>
```

Comment:
Please make your own plugin instead of piling on top of the ZipTree Nodestore.
You can start by copying ZipTree and then making changes.

I prefer some code duplication instead of an ifdef-forest.